quantum/rgb_matrix/post_config.h

```diff
@@ -0,0 +1,53 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+// framebuffer
+#if defined(ENABLE_RGB_MATRIX_TYPING_HEATMAP)
+#    define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+#endif
+#if defined(ENABLE_RGB_MATRIX_DIGITAL_RAIN)
+#    define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+#endif
+
+// reactive
+#if defined(ENABLE_RGB_MATRIX_SOLID_REACTIVE_SIMPLE)
```

Comment:
the only issue here, is that these effects can be triggered on key up or key down.  This basically forces it to be keydown.

Maybe wrap it in `#if !defined(RGB_MATRIX_KEYPRESSES) && !defined(RGB_MATRIX_KEYRELEASES)`?

Reply:
Nope it does not, as the logic is <https://github.com/qmk/qmk_firmware/blob/master/quantum/rgb_matrix/rgb_matrix.c#L219>.

`_MATRIX_KEYRELEASES` takes precedence, and defining both does not matter.

Reply:
Ah, it's been a while.  So yeah, should be fine, then.